debian/ceph-common.install

```diff
@@ -24,6 +24,8 @@ usr/bin/ceph-post-file
 usr/sbin/mount.ceph sbin
 usr/lib/ceph/compressor/*
 usr/lib/ceph/crypto/* [amd64]
+usr/lib/python*/dist-packages/ceph-*.egg-info
+usr/lib/python*/dist-packages/ceph
```

Comment:
Is it expected that the DEB package contains a different set of files than RPM?

Reply:
```
$ rpm --eval %python3_sitelib
/usr/lib/python3.6/site-packages
```

Reply:
it works! otherwise teuthology would have failed. But yes, it's dist-packages.